src/main/java/org/example/homework3/Task2.java

```diff
@@ -0,0 +1,19 @@
+package org.example.homework3;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int a = 210;
+        int b = 15;
+        int min;
+        int max;
+        if (a > b) {

```

Comment:
Try to use the ternary operator here for defining min and max.
Solution is absolutely correct, but we didn't learn if-else construction, yet

Reply:
I have refactored task2 using the ternary operator. I have squashed upates into one commit.

Reply:
Feel free to merge, PR is already approved